src/DependencyInjection/DoctrineExtension.php

```diff
@@ -1163,7 +1171,31 @@ public function getConfiguration(array $config, ContainerBuilder $container): Co
 
     protected function getMetadataDriverClass(string $driverType): string
     {
-        return '%' . $this->getObjectManagerElementName('metadata.' . $driverType . '.class') . '%';
+        switch ($driverType) {
+            case 'driver_chain':
+                return MappingDriverChain::class;
+
+            case 'annotation':
+                return AnnotationDriver::class;
```

Comment:
this should probably check for existence, as this driver does not exist in doctrine/orm 3.x

Reply:
Since the `doctrine.orm.metadata.annotation.class` parameter still exists this should somewhat keep the current behavior?

Or should I throw an exception if the class does not exist?

Reply:
yes please throw exception informing the user this driver is only available for doctrine/orm 2.x